doc/changelog/04-tactics/12986-master+ordering-notation-by-precision.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  Use of notations for printing now (experimentally) gives preference
```

Comment:
I don't like this "experimentally". What are you trying to tell the user?
IMO it is better to remove it.

Reply:
@gares This is part of the base PR (#12986) and in this PR, I've already suggested a change that Hugo has implemented. See https://github.com/coq/coq/pull/12986/files#diff-e49d5d4ee2dcdca9f4f5b8020ae461787e009050f48333a844cf5898f0e2b42d. This PR is not up-to-date with the one it depends on.

Reply:
(Probably because the change was applied in a separate commit that @herbelin forgot to pull.)

Reply:
To digress, I've seen this problem before and this can even lead to suggestions being lost upon force-pushing (always using `--force-with-lease` instead of `--force` can prevent this).

Reply:
> (Probably because the change was applied in a separate commit that @herbelin forgot to pull.)

Just because there is no automatic rebasing of a branch on another when the other changes... (I always use `--force-with-lease` for quite some time now).

I'm ok with removing `experimental` otherwise. No objections?

Reply:
> I'm ok with removing experimental otherwise. No objections?

But this means a change in the other PR (and then, I remove also the "Please report any issue", ok?)